lib/raven/configuration.rb

```diff
@@ -210,7 +210,10 @@ def detect_release_from_heroku
     end
 
     def detect_release_from_capistrano
-      File.read(File.join(project_root, 'REVISION')).strip rescue nil
+      version = File.read(File.join(project_root, 'REVISION')).strip rescue nil
+
+      # Capistrano 3+
+      version || File.open(File.join(Rails.root, '..', 'revisions.log')).to_a.last.strip.sub(/.*as release ([0-9]+).*/, '\1') rescue nil
```

Comment:
also, I'm not a Capistrano user, but [this blog post](http://building.wanelo.com/2014/03/31/capistrano-you-have-changed.html) suggests the revisions.log file lives right in the application root?